Writerside/topics/ChiselMethod.md

```diff
@@ -1,3 +1,24 @@
 # The Chisel Method
 
-_To be written_
\ No newline at end of file
+The Chisel method for writing parsers allows to build parsers consistently using the same approach which allows to achieve a consistency on the architectural decisions and enables developers to get familiar with a project easily.
```

Comment:
Here parsers is repeated. We may write "The Chisel method allows to build parsers consistently using the same approach..."